internal/api/kubernetes/client.go

```diff
@@ -26,6 +26,7 @@ const (
 )
 
 // Client is an interface representing a high-level client to get and modify Kubernetes resources.
+
 type Client interface {
 	// GetNamespaceAnnotations gets the annotations of the workload's namespace
```

Comment:
```suggestion
// Client is an interface representing a high-level client to get and modify Kubernetes resources.
type Client interface {
	// GetNamespaceAnnotations gets the annotations of the workload's namespace
```
I think my prior suggestion for some reason didn't delete that line